src/subcommands/clone/mod.ts

```diff
@@ -0,0 +1,13 @@
+import { Command } from "cliffy";
+import errorHandler from "@/error-handler.ts";
+
+import { GlobalOptions } from "@/args.ts";
+import action from "./action.ts";
+
+const command = new Command<GlobalOptions>()
+  .description("Clone a Jet project.")
+  .arguments("<projectId>")
```

Comment:
支持指定 directory 方便不呀？

```suggestion
  .arguments("<projectId> [<directory>]")
```